project/src/store/action.js

```diff
@@ -1,44 +1,37 @@
+import { createAction } from '@reduxjs/toolkit';
+
 export const ActionType = {
   SET_CITY: 'main/setCity',
   REDIRECT_TO_ROUTE: 'main/redirectToRoute',
   REQUIRED_AUTHORIZATION: 'user/requiredAuthorization',
   LOAD_OFFERS: 'data/loadOffers',
   SET_USER: 'user/setUser',
   LOGOUT: 'user/logout',
-  SET_OFFER_DATA: 'data/setOfferData',
   UPDATE_OFFER: 'data/updateOffer',
 };
 
-export const ActionCreator = {
-  setCity: (city) => ({
-    type: ActionType.SET_CITY,
-    payload: city,
-  }),
-  requireAuthorization: (status) => ({
-    type: ActionType.REQUIRED_AUTHORIZATION,
-    payload: status,
-  }),
-  loadOffers: (offers) => ({
-    type: ActionType.LOAD_OFFERS,
-    payload: offers,
-  }),
-  redirectToRoute: (url) => ({
-    type: ActionType.REDIRECT_TO_ROUTE,
-    payload: url,
-  }),
-  setUser: (user) => ({
-    type: ActionType.SET_USER,
-    payload: user,
-  }),
-  logout: () => ({
-    type: ActionType.LOGOUT,
-  }),
-  setOfferData: (offerData) => ({
-    type: ActionType.SET_OFFER_DATA,
-    payload: offerData,
-  }),
-  updateOffer: (offer) => ({
-    type: ActionType.UPDATE_OFFER,
-    payload: offer,
-  }),
-};
+export const setCity = createAction(ActionType.SET_CITY, (city) => ({
+  payload: city,
+}));
+
+export const requireAuthorization = createAction(ActionType.REQUIRED_AUTHORIZATION, (status) => ({
+  payload: status,
+}));
```

Comment:
Вот так имею в виду:
```suggestion
export const requireAuthorization = createAction(ActionType.REQUIRED_AUTHORIZATION);
```